src/cmd/cli/command/commands.go

```diff
@@ -198,46 +199,22 @@ func SetupCommands(version string) {
 	RootCmd.AddCommand(configCmd)
 	RootCmd.AddCommand(restartCmd)
 
-	// Compose Command
-	// composeCmd.Flags().Bool("compatibility", false, "Run compose in backward compatibility mode"); TODO: Implement compose option
-	// composeCmd.Flags().String("env-file", "", "Specify an alternate environment file."); TODO: Implement compose option
-	// composeCmd.Flags().Int("parallel", -1, "Control max parallelism, -1 for unlimited (default -1)"); TODO: Implement compose option
-	// composeCmd.Flags().String("profile", "", "Specify a profile to enable"); TODO: Implement compose option
-	// composeCmd.Flags().String("project-directory", "", "Specify an alternate working directory"); TODO: Implement compose option
-	composeUpCmd.Flags().Bool("tail", false, "tail the service logs after updating") // obsolete, but keep for backwards compatibility
-	_ = composeUpCmd.Flags().MarkHidden("tail")
-	composeUpCmd.Flags().Bool("force", false, "force a build of the image even if nothing has changed")
-	composeUpCmd.Flags().BoolP("detach", "d", false, "run in detached mode")
-	composeCmd.AddCommand(composeUpCmd)
-	composeCmd.AddCommand(composeConfigCmd)
-	composeDownCmd.Flags().Bool("tail", false, "tail the service logs after deleting") // obsolete, but keep for backwards compatibility
-	composeDownCmd.Flags().BoolP("detach", "d", false, "run in detached mode")
-	_ = composeDownCmd.Flags().MarkHidden("tail")
-	composeCmd.AddCommand(composeDownCmd)
-	composeStartCmd.Flags().Bool("force", false, "force a build of the image even if nothing has changed")
-	composeCmd.AddCommand(composeStartCmd)
-	composeCmd.AddCommand(composeRestartCmd)
-	composeCmd.AddCommand(composeStopCmd)
-	// composeCmd.AddCommand(getServicesCmd) // like docker compose ls
-	RootCmd.AddCommand(composeCmd)
-
+	RootCmd.AddCommand(setupComposeCommand())
 	// Add up/down commands to the root as well
-	// RootCmd.AddCommand(composeDownCmd)
-	// RootCmd.AddCommand(composeUpCmd)
-	// RootCmd.AddCommand(composeStartCmd)
-	// RootCmd.AddCommand(composeRestartCmd)
-	// RootCmd.AddCommand(composeStopCmd)
+	RootCmd.AddCommand(makeComposeDownCmd())
+	RootCmd.AddCommand(makeComposeUpCmd())
+	RootCmd.AddCommand(makeComposeStartCmd())
```

Comment:
lets leave start/stop/restart off root for now, since they don't technically do the right thing